libnavigation-ui/src/main/java/com/mapbox/navigation/ui/map/NavigationMapboxMap.java

```diff
@@ -117,16 +118,23 @@ public NavigationMapboxMap(@NonNull MapView mapView,
                              @Nullable String routeBelowLayerId) {
     this.mapView = mapView;
     this.mapboxMap = mapboxMap;
-    initializeLocationComponent(mapView, mapboxMap);
     initializeMapPaddingAdjustor(mapView, mapboxMap);
     initializeNavigationSymbolManager(mapView, mapboxMap);
     initializeMapLayerInteractor(mapboxMap);
     initializeRoute(mapView, mapboxMap, routeBelowLayerId);
-    initializeCamera(mapboxMap, locationComponent);
-    initializeLocationFpsDelegate(mapboxMap, locationComponent);
     initializeArrivalExperience(mapboxMap, mapView);
+    initializeCamera(mapboxMap);
   }
 
+  public void initializeLocationComponent() {
+    setupLocationComponent(mapView, mapboxMap);
+    initializeLocationFpsDelegate(mapboxMap, locationComponent);
+  }
+
+  public void enterNavigationMode() {
```

Comment:
Do we need to expose this? The user can always call `NavigationMapboxMap#updateCameraTrackingMode` and `NavigationMapboxMap#updateLocationLayerRenderMode`. I'd prefer to limit the API surface.

Reply:
My motivation for doing this was to abstract those details. It's not intuitive that a developer would know those two methods have to be called and when. That said, I solicited feedback because I knew I was introducing a new external method.